### PR TITLE
UWC : Added fix for scans

### PR DESCRIPTION
**Pull request by aditisa1**
1) Made changes according to DBS report

Signed-off-by: aditisa1 <aditi.saxena@intel.com>

### CODE REVIEW

File: MQTT/Dockerfile [hunk 1]
```diff
@@ -32,5 +32,5 @@ COPY mosquitto_dev.conf /etc/mosquitto/mosquitto_dev.conf
 
 COPY mqtt_broker_start.sh /
 RUN chmod +x mqtt_broker_start.sh
-
+HEALTHCHECK CMD ["/bin/sh","-c", "ps -C mosquitto_sub >/dev/null && echo Running || echo Not running; exit 1"]
```
Comment (nagdeepgk) [inline MQTT/Dockerfile:35]: We had a healthcheck even earlier right ?

File: Vendor_Apps/EmbPublisher/docker-compose.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ services:
     container_name: emb_publisher
     restart: unless-stopped
     hostname: emb_publisher
-    user: root
+    read_only: true
```
Comment (nagdeepgk) [inline Vendor_Apps/EmbPublisher/docker-compose.yml:25]: This readme=true was added now Why ?

File: kpi-tactic/Dockerfile [hunk 3]
```diff
@@ -21,13 +21,35 @@
...
 ARG UBUNTU_IMAGE_VERSION
 ARG UWC_VERSION
+ARG CMAKE_INSTALL_PREFIX
 FROM uwc_common:$UWC_VERSION as uwc_common
 LABEL description="KPI Tactic Application"
 
 # Build KPI Tactic application
 FROM ia_common:$EII_VERSION as kpitactic
 WORKDIR /
+# Install libzmq
```
Comment (nagdeepgk) [inline kpi-tactic/Dockerfile:31]: looks like rebase PRs changes have ocm ehere too. Some issue with git handling.

File: kpi-tactic/Dockerfile [hunk 4]
```diff
@@ -82,6 +105,6 @@ RUN chown -R ${EII_UID}:${EII_UID} /opt/intel/config \
 
 #set the environment variable
 ENV LD_LIBRARY_PATH ${APP_WORK_DIR}:/usr/local/lib
-
+HEALTHCHECK CMD ["/bin/sh","-c", "ps -C KPIAPP >/dev/null && echo Running || echo Not running; exit 1"]
```
Comment (nagdeepgk) [inline kpi-tactic/Dockerfile:108]: healthcheck was there earlier too ?

**Comment by aditisa1** [comment]
@nagdeepgk 
healthcheck and Read_only are added as a fix for DBS scan 
in current report we got issue in Healthcheck too (previously we didn't get)
we got it as we were missing health check in Dockerfile

**Comment by aditisa1** [comment]
Closing this PR as it contains rebase changes 
reopened https://github.com/open-edge-insights/uwc/pull/64

**Comment by aditisa1** [comment]
Created following PR for this
https://github.com/open-edge-insights/uwc/pull/68/files#

**Pull request closed without merging**